src/GitHub.App/SampleData/PullRequestDetailViewModelDesigner.cs

```diff
@@ -70,6 +70,7 @@ public PullRequestDetailViewModelDesigner()
         public IPullRequestModel Model { get; }
         public string SourceBranchDisplayName { get; set; }
         public string TargetBranchDisplayName { get; set; }
+        public bool IsBusy { get; }
```

Comment:
`IHasBusy` is already on `IPullRequestDetailViewModel` so is already being implemented.